pkg/pushsync/pb/pushsync.proto

```diff
@@ -15,4 +15,6 @@ message Delivery {
 
 message Receipt {
   bytes Address = 1;
+  bytes StorerAddress = 2;
```

Comment:
I think that the addition of the storer address in the message is not necessary, since the address can be recovered from the signature

Reply:
This check was meant to prevent nodes from signing on behalf of other nodes, eg. public key in the signature being different from the address of the node.

After this, we still have the problem of uncertainty about the signer being the actual storer.

Reply:
This does not provide much more protection. If you spoof a new bogus key to sign the message in another node's name, what's preventing you from setting the address field on the message too?

Reply:
Hmm. The node may indeed create a bogus key and a node address, and sign the receipt with the address and public key being the same. It does not provide much protection, like you mentioned. 👍